src/main/java/com/crud/tasks/service/SimpleEmailService.java

```diff
@@ -22,21 +22,32 @@
     @Autowired
     private MailCreatorService mailCreatorService;
 
-    public void send(final Mail mail) {
+    public void send(final Mail mail, EmailType emailType) {
```

Comment:
Nie rozumiem idei użycia tego `enum`-a. Jest tylko po to, żeby zalogować jeden albo drugi komunikat. Trzeba pomyśleć o czymś bardziej generycznym - jak będzie w systemie 50 różnych maili, to będziesz miała enuma z 50 wartościami i 50 `if...else`?